kategory-core/src/main/kotlin/kategory/data/State.kt

```diff
@@ -1,13 +1,72 @@
 package kategory
 
+/**
+ * Convenience object to make working with `StateT<IdHK, S, A>` more elegant.
+ */
 object State {
```

Comment:
Can we `typealias State<S, A> = StateT<IdHK, S, A>` and use the alias in all return types? That we we can define many of these methods as extensions to the alias which would allow users to not have to see the underlying StateT impl.

Reply:
`typealias State<S, A>` conflicts with `object State`.